inst/include/frogger.hpp

```diff
@@ -1,16 +1,21 @@
 #pragma once
 
 #include "demographic_projection.hpp"
+#include "hiv_demographic_projection.hpp"
```

Comment:
The population that it is run on, HIV demographic projection runs on hiv population data, and hiv population stratified by disease stage and art population stratified by disease stage and treatment stage.

The demographic projection `run_ageing_and_mortality` and `run_hiv_ageing_and_mortality` are quite similar you are right. Only difference is the HIV one doesn't remove the natural deaths from the hiv population until later - I think because it uses the hiv population to calculate the stratified populations.

The migration code is quite different because the demographic projection part calculates a migration rate as a proportion of the population size, this is then reused in the HIV demographic projection